jetpack/chart/utils.py

```diff
@@ -127,9 +187,18 @@ def breathe(factor=0.05, **kwargs):
     ax.spines['left'].set_bounds(ya, yb)
 
     nospines(**kwargs)
+    tickdir(direction=direction, **kwargs)
 
     return ax
 
+@axwrapper
+def tickdir(direction='out', **kwargs):
```

Comment:
Could you make the direction a required argument, instead of having it default to `out`?

Out isn't an obvious default